setup.pp

```diff
@@ -96,13 +96,22 @@
   'pygame',
   'pyserial',
   'requests',
+  'cherrypy',
 ]
 
 python::pip { $python_packages:
   pip_provider => 'pip3',
   require      => Package[ $main_packages, $piweatherrock_packages, ],
 }
 
+# Run upgrade script to import current config values to new config file
+exec { 'import config':
+  path    => '/bin:/usr/bin',
+  command => 'python3 /home/pi/PiWeatherRock/scripts/upgrade.py',
+  unless  => 'grep "0.0.13" config.json',
+  notify  => Service['PiWeatherRock.service'],
+}
```

Comment:
This fixes the config issue. Otherwise the file is owned by root.

```suggestion
exec { 'import config':
  user    => 'pi',
  path    => '/bin:/usr/bin',
  command => 'python3 /home/pi/PiWeatherRock/scripts/upgrade.py',
  unless  => 'grep "0.0.13" config.json',
  notify  => Service['PiWeatherRock.service'],
}
```

Reply:
I think I got all of these changes implemented.